### DISCUSSION TITLE
Google API Key?

### DISCUSSION THREAD

**Discussion by zebz213** [DevYukine/rustfoil#14] [Q&A] [answered]
What exactly would this be used for? 
Does it allow users to download the drive files from TFL without needing oauth credentials?

**Comment by DevYukine** [comment, marked as answer]
This is per Tinfoil spec, it is used to download from google drive authenticated but it doesn't allow for replacing OAuth (its not used for permission managment like OAuth is), its just making authenticated calls which have a higher quota compared to non authenticated calls to files so its less likely to quota a file if a lot of different people download it authenticated compared to non authenticated.
